code/modules/projectiles/projectile/bullets.dm

```diff
@@ -164,11 +164,16 @@
 	sharp = 0
 	incinerate = 10
 
+/obj/item/projectile/bullet/sabot
+	name = "sabot"
+	damage = 30
```

Comment:
30 seems slightly excessive for damage? It's nearly laser rifle level *and* it's a ballistic caliber. I would make it 25 personally or less, as I'm kind of afraid of these being used in normal situations.

Reply:
Well, the damage is identical to the .45 and is inferior in every regard but armor penetration. I think it's fine, but it can be changed if needed.